asapdiscovery/simulation/scripts/MproP0009.py

```diff
@@ -39,18 +32,67 @@ def get_args():
         default = "/data/chodera/lemonsk/asap-datasets/prepped_mpro_P0009/",
         help="Output simulation directory."
     )
+    parser.add_argument(
+        "-l",
+        "--ligand",
+        type = str,
+        default = "/data/chodera/lemonsk/covid-moonshot-ml/asapdiscovery/simulation/tests/inputs/MAT-POS-f2460aef-1.sdf",
+        help = "Ligand"
+    )
+    parser.add_argument(
+        "-p",
+        "--protein",
+        type = str,
+        default = "/data/chodera/lemonsk/asap-datasets/openmm_setup_processed/prepped_receptor_0-processed.pdb",
+        help = "Protein-receptor"
+    )
     args = parser.parse_args()
     return args
 
 args = get_args()
 
+from rdkit import Chem
+rdkitmol = Chem.SDMolSupplier(args.ligand)[0]
+rdkitmolh = Chem.AddHs(rdkitmol, addCoords=True);
+# ensure the chiral centers are all defined
+Chem.AssignAtomChiralTagsFromStructure(rdkitmolh);
+from openff.toolkit.topology import Molecule
+ligand_mol = Molecule(rdkitmolh);
+
+# Initialize a SystemGenerator
+from openmm import unit, app
+from openmmforcefields.generators import SystemGenerator
+forcefield_kwargs = {'constraints': app.HBonds, 'rigidWater': True, 'removeCMMotion': False, 'hydrogenMass': 4*unit.amu }
+periodic_forcefield_kwargs = {'nonbondedMethod': app.PME}
+system_generator = SystemGenerator(
+    forcefields=['amber/ff14SB.xml', 'amber/tip3p_standard.xml'],
+    small_molecule_forcefield='openff-1.3.1',
+    molecules=[ligand_mol], cache='cache.json',
+    forcefield_kwargs=forcefield_kwargs, periodic_forcefield_kwargs=periodic_forcefield_kwargs);
+
+# Use Modeller to combine the protein and ligand into a complex
+from openmm.app import PDBFile
```

Comment:
your imports are really messy!
You're importing something from openmm.app multiple times. I know John likes lazy imports but I think this could be the problem

Reply:
Yep, my imports are very messy lol. I was still polishing but went ahead and pushed. I was initially trying to fix the star imports, but I was leaving a lot of things out hence the repeated imports from the different components. Yes, this needs to be cleaned up a ton, but at this moment, from what I can tell its not causing any errors